api/src/main/java/io/smallrye/stork/api/config/LoadBalancerName.java

```diff
@@ -0,0 +1,13 @@
+package io.smallrye.stork.api.config;
+
+import static java.lang.annotation.ElementType.TYPE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(TYPE)
+public @interface LoadBalancerName { /* mstodo rename to LBType? */
```

Comment:
+1, or rename "type" to "name"

Reply:
I'm especially interested in your opinion on the format: https://github.com/smallrye/smallrye-stork/pull/195/files#diff-1224f5ee4e7cf65a378a09a023396a12ff277fc7539783ae7afc332a7c150aa5R9-R12

and the things I listed in the PR description

Reply:
that unfortunately causes more mess :( I'll rename to LoadBalancerType